.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,48 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or the definition README at
+// https://github.com/microsoft/vscode-dev-containers/tree/master/containers/python-3 or the
+// devcontainer docu https://code.visualstudio.com/docs/remote/containers#_devcontainerjson-reference
+{
+  "name": "sihl",
+  "dockerComposeFile": "./docker-compose.yml",
+  "service": "dev",
+  "runServices": [
+    "dev",
+    "postgres",
+    "mariadb"
+    // "adminer" // uncomment if adminer should run to see the database
+  ],
+  "workspaceFolder": "/workspace",
+  "remoteEnv": {
+    "DATABASE_URL": "postgres://admin:password@postgres:5432/dev",
+    "DATABASE_URL_TEST_MARIADB": "mariadb://admin:password@mariadb:5432/dev",
+    "DATABASE_URL_TEST_POSTGRES": "postgres://admin:password@postgres:5432/dev",
```

Comment:
@mabiede Let's use the full name to be consistent `DATABASE_URL_TEST_POSTGRESQL`